internal/controllers/packages/object_deployment_status_reconciler.go

```diff
@@ -43,6 +44,16 @@ func (r *objectDeploymentStatusReconciler) Reconcile(
 		meta.SetStatusCondition(packageObj.GetConditions(), *packageProgressingCond)
 	}
 
+	objDepPausedCond := meta.FindStatusCondition(*objDep.GetConditions(), corev1alpha1.ObjectDeploymentPaused)
+	if objDepPausedCond != nil && objDepPausedCond.ObservedGeneration == objDep.ClientObject().GetGeneration() &&
+		objDepPausedCond.Status == metav1.ConditionTrue {
+		packagePausedCond := objDepPausedCond.DeepCopy()
+		packagePausedCond.ObservedGeneration = packageObj.ClientObject().GetGeneration()
+		meta.SetStatusCondition(packageObj.GetConditions(), *packagePausedCond)
+	} else {
+		meta.RemoveStatusCondition(packageObj.GetConditions(), corev1alpha1.PackagePaused)
```

Comment:
I don't think we should remove the status condition here. We should keep it and explicitly set it to false.

Reply:
ah whatever - i don't need to be nitpicky